di/datasets.csv

```diff
@@ -0,0 +1,57 @@
+ID,Group (datasets of the same group are similar*),Project,Title (with Hyperlink),Description (in brief),Metadata URLs,Methodology (URL to methodology descriptions),Methodology (brief description),Accuracy,Completeness,Level,Spatial Granularity,Identifier,Identifier Type,Creator,Object,Publisher,Publication Date,Publication Year,Temporal Granularity,Time references (time data refers),URLs,Content (keywords),Origin,Geographical extension,Projection system,Access conditions,License,Terms of use,Availability,Resource type,Data format,"Size of file (raw, compressed in parentheses)",Other relevant information

```

Comment:
latin1 is not really a recommended encoding for files, could you translate this file to use utf-8 instead ?